app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="Master PDF Editor is a complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
```

Comment:
`-> ${P}.tar.gz`

Reply:
Why? The distfile name is properly versioned already. Renaming it gives no benefit. Please stop forcing that everywhere for no reason.